code/game/atoms_movable.dm

```diff
@@ -4,6 +4,7 @@
 #if DM_VERSION >= 512
 #error HEY, PIXEL_SCALE probably exists now, remove this gross ass shim.
 #endif
+#endif

```

Comment:
I know ```#endif```s aren't necessary in all cases, cos oranges did an 
```#if DM_VERSION > 512
...
#elseif
```

with no ```#endif```, the reference also says ```#if``` goes to the next ```#elseif,#endif```, which implies they don't need to match amounts.